src/open_prime_hunters_rando/overlays.py

```diff
@@ -26,7 +31,7 @@ def patch_overlays(rom: NintendoDSRom) -> None:
         for offset_value in offset_values:
             offset = offset_value["offset"]
             value = offset_value["value"]
-            overlay.data[offset] = value
+            overlay.data[offset : offset + len(value)] = value
```

Comment:
i dont quite get what this is supposed to be doing. i assumed its related to the `to_bytes` change you did?